Cargo.toml

```diff
@@ -24,6 +24,7 @@ async-std = { version = "1.5.0", optional = true }
 async-trait = "0.1.24"
 log = "0.4.8"
 tokio = { version = "0.2.11", optional = true, default-features = false }
+futures = "0.3"
```

Comment:
I hope its ok to give some feedback. @GopherJ told me not to add `futures` to `[dev-dependencies]` so what I did instead was [this](https://github.com/casbin/casbin-rs/blob/master/benches/casbin.rs#L4-L18). If this is a C compatibility thing ignore me, feel free to ignore me either way :smile:!

Reply:
Yes, you are right, no need to add `futures` here, we can use `async_std::task::block_on` instead. I will make a change.